<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PSFEditorDialog</class>
 <widget class="QDialog" name="PSFEditorDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>774</width>
    <height>727</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Point-Spread Function Editor</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <property name="leftMargin">
    <number>12</number>
   </property>
   <property name="topMargin">
    <number>6</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>6</number>
   </property>
   <item>
    <widget class="QSplitter" name="gui_WindowSplitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QWidget" name="gui_ControlPanel" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>400</width>
        <height>16777215</height>
       </size>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="QSplitter" name="gui_ControlPanelSplitter">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <widget class="QGroupBox" name="gui_PSFListGroupBox">
          <property name="maximumSize">
           <size>
            <width>400</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="title">
           <string>Point-Spread Functions</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <property name="spacing">
            <number>-1</number>
           </property>
           <property name="topMargin">
            <number>12</number>
           </property>
           <item>
            <widget class="QListView" name="gui_PSFListView"/>
           </item>
           <item>
            <widget class="QPushButton" name="gui_AddCalculatedGaussianPSFButton">
             <property name="text">
              <string>Add Calculated Gaussian PSF</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="gui_AddCalculatedWidefieldPSFButton">
             <property name="text">
              <string>Add Calculated Widefield PSF</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="gui_ImportPSFButton">
             <property name="text">
              <string>Import PSF...</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="gui_DeletePSFButton">
             <property name="text">
              <string>Delete PSF</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QGroupBox" name="gui_PSFParametersGroupBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>400</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="title">
           <string>Point-Spread Function Parameters</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QTableView" name="gui_PSFParametersView">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="autoScrollMargin">
              <number>4</number>
             </property>
             <property name="alternatingRowColors">
              <bool>true</bool>
             </property>
             <property name="selectionMode">
              <enum>QAbstractItemView::NoSelection</enum>
             </property>
             <attribute name="horizontalHeaderDefaultSectionSize">
              <number>160</number>
             </attribute>
             <attribute name="horizontalHeaderStretchLastSection">
              <bool>true</bool>
             </attribute>
             <attribute name="verticalHeaderVisible">
              <bool>false</bool>
             </attribute>
             <attribute name="verticalHeaderDefaultSectionSize">
              <number>16</number>
             </attribute>
             <attribute name="verticalHeaderDefaultSectionSize">
              <number>16</number>
             </attribute>
             <attribute name="horizontalHeaderStretchLastSection">
              <bool>true</bool>
             </attribute>
             <attribute name="horizontalHeaderDefaultSectionSize">
              <number>160</number>
             </attribute>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>Point-Spread Function Display</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <item>
            <widget class="QWidget" name="widget_2" native="true">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout">
              <property name="spacing">
               <number>3</number>
              </property>
              <property name="margin">
               <number>0</number>
              </property>
              <item>
               <widget class="QLabel" name="gui_OrientationLabel">
                <property name="text">
                 <string>Orientation:</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QToolButton" name="gui_XPlusButton">
                <property name="text">
                 <string>+X</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QToolButton" name="gui_XMinusButton">
                <property name="text">
                 <string>-X</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QToolButton" name="gui_YPlusButton">
                <property name="text">
                 <string>+Y</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QToolButton" name="gui_YMinusButton">
                <property name="text">
                 <string>-Y</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QToolButton" name="gui_ZPlusButton">
                <property name="text">
                 <string>+Z</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QToolButton" name="gui_ZMinusButton">
                <property name="text">
                 <string>-Z</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="gui_HorizontalSpacer2">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>0</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QWidget" name="widget_3" native="true">
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <property name="margin">
               <number>0</number>
              </property>
              <item>
               <widget class="QLabel" name="gui_MinLevelLabel">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string>Min level:</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="gui_MinLevelEdit">
                <property name="text">
                 <string>0</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QSlider" name="gui_MinLevelSlider">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QWidget" name="widget_4" native="true">
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <property name="margin">
               <number>0</number>
              </property>
              <item>
               <widget class="QLabel" name="gui_MaxLevelLabel">
                <property name="text">
                 <string>Max level:</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="gui_MaxLevelEdit">
                <property name="text">
                 <string>1</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QSlider" name="gui_MaxLevelSlider">
             <property name="value">
              <number>99</number>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="gui_RescaleButton">
             <property name="text">
              <string>Scale to Full Dynamic Range</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>0</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="widget" native="true">
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="QVTKWidget" name="gui_PSFDisplayQvtkWidget" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDialogButtonBox" name="gui_ButtonBox">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="standardButtons">
          <set>QDialogButtonBox::Ok</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QVTKWidget</class>
   <extends>QWidget</extends>
   <header>QVTKWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>gui_ButtonBox</sender>
   <signal>accepted()</signal>
   <receiver>PSFEditorDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>gui_ButtonBox</sender>
   <signal>rejected()</signal>
   <receiver>PSFEditorDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
